docs/HowTo/Use-Privacy/Use-OnChainPrivacy.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Use Onchain Privacy
+---
+
+# Using Onchain Privacy Groups 
+
+Use the [web3.js-eea library](https://github.com/PegaSysEng/web3js-eea) to create and update 
+membership of onchain privacy groups. 
+
+!!! important 
+    Onchain privacy groups are an early access feature in v1.4. Do not use in production networks. 
+
+    The onchain privacy group interfaces may change between v1.4 and v1.5. There may not be an upgrade 
+    path from onchain privacy groups created using v1.4 to enable use of onchain privacy group functionality 
+    in future versions. 
+
+    We do not recommend creating onchain privacy groups in a chain with existing 
+    [offchain privacy groups](../../Concepts/Privacy/Privacy-Groups.md). 
+    
+## Enabling onchain privacy groups 
+
+Use the [`--privacy-onchain-groups-enabled`](../../Reference/CLI/CLI-Syntax.md#privacy-onchain-groups-enabled)
+command line option to enable onchain privacy groups. 
+    
+## Simple onchain privacy group example 
+
+To create and find an onchain privacy group using the [web3.js-eea library](https://github.com/PegaSysEng/web3js-eea): 
+
+1. Update the `example/keys.js` file to match your network configuration. 
+
+2. Run: 
+
+    ```
+    node simpleOnChainPrivacy.js
+    ```
+
+    Two transactions are created and mined. The first transaction initiates the onchain privacy group
+    and the second transaction adds the members. 
+
```

Comment:
I think it's only 1 transaction. This initiates a privacy group with 2 memebers.

Reply:
I was misreading the output here.  Updating.